pkg/porter/explain_test.go

```diff
@@ -224,6 +231,13 @@ func TestExplain_generatePrintableBundleOutputs(t *testing.T) {
 				Definition: "string",
 			},
 		},
+		Custom: map[string]interface{}{
```

Comment:
Same, we only need this extra testdata in the test where we are asserting on the porter version in the explain output.